docs/data-collection.md

```diff
@@ -162,6 +162,10 @@ For the OpenShift 4 Developer Preview we will be sending back these exact attrib
   // cam_app_workload_migrations tracks number of app workload migrations
   // by current state. Tracked migration states are idle, running, completed, and failed.
   '{__name__="cam_app_workload_migrations"}',
+  // apiserver_current_inflight_requests refers to maximal number of currently used
+  // inflight request limit of an apiserver per request kind in last second.
+  // We want to have an idea of how loaded our api server(s) are globally.
+  '{__name__="sum(apiserver_current_inflight_requests) by(requestKind)"}',
```

Comment:
Can we rename the label `requestKind` to just `kind`?

Reply:
This metric is exposed by upstream `kube-apiserver`.

Reply:
this doesn't work, you need to have a separate recording rule that aggregates this in your namespace and then just whitelist the recording rule here